doc/classes/Tree.xml

```diff
@@ -31,6 +31,7 @@
 		[/csharp]
 		[/codeblocks]
 		To iterate over all the [TreeItem] objects in a [Tree] object, use [method TreeItem.get_next] and [method TreeItem.get_first_child] after getting the root through [method get_root]. You can use [method Object.free] on a [TreeItem] to remove it from the [Tree].
+		[b]Recursive folding:[/b] If [member enable_recursive_folding] is [code]true[/code] holding down shift while clicking the fold arrow collapses or uncollapses the [TreeItem] and all its descendants.
```

Comment:
I think this can be moved to `enable_recursive_folding` description. Incremental search is described here, because it doesn't have dedicated member.

Reply:
What do you think about:
If [code]true[/code], recursive folding is enabled for this [Tree] and holding down shift while clicking the fold arrow collapses or uncollapses the [TreeItem] and all its descendants.

Reply:
Capitalize shift. Also feels like the `and` is not necessary here, but not sure.

Reply:
If [code]true[/code], recursive folding is enabled for this [Tree]. Holding down Shift while clicking the fold arrow collapses or uncollapses the [TreeItem] and all its descendants.